docs/deploying_csi_vsphere_with_rbac.md

```diff
@@ -117,52 +120,114 @@ Create the secret by running the following command:
+You can find the RBAC roles and bindings required by the CSI node manager in [vsphere-csi-node-rbac.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-node-rbac.yaml).
 
 To apply them to your Kubernetes cluster, run the following command:
 
 ```bash
-[k8suser@k8master ~]$ kubectl create -f csi-role-bindings.yaml
+[k8suser@k8master ~]$ kubectl create -f vsphere-csi-node-rbac.yaml
 ```
 
-#### 5. Deploy `csi-vsphere`
-
-You have two options for deploying `csi-vsphere`. It can be deployed either as a simple Pod or in a DaemonSet. There really isn't much difference between the two other than the DaemonSet will do leader election and the Pod will just assume to be the leader.
+#### 5. Deploy `csi-vsphere-controller`
 
 **IMPORTANT NOTES:**
-- Deploy either as a Pod or in a DaemonSet, but *DO NOT* deploy both.
-- The YAML to deploy as a Pod or a DaemonSet assume that your Kubernetes cluster was deployed using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). If you deployed your cluster using alternate means, you will need to modify the either of the YAML files in order to provided necessary files or paths based on your deployment.
-
-##### Deploy `csi-vsphere` as a Pod
+- The YAML to deploy CSI controller assumes that your Kubernetes cluster was deployed using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). If you deployed your cluster using alternate means, you will need to modify the either of the YAML files in order to provided necessary files or paths based on your deployment.
 
-The YAML to deploy `csi-vsphere` as a Pod can be found in [vsphere-csi-pod.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-pod.yaml).
+The YAML to deploy `csi-vsphere-controller` can be found in [vsphere-csi-controller-ss.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-controller-ss.yaml).
 
 Run the following command:
 
 ```bash
-[k8suser@k8master ~]$ kubectl create -f vsphere-csi-pod.yaml
+[k8suser@k8master ~]$ kubectl create -f vsphere-csi-controller-ss.yaml
 ```
 
-##### Deploy `csi-vsphere` as a DaemonSet
+#### 6. Deploy `csi-vsphere-node`
 
-The YAML to deploy `csi-vsphere` as a DaemonSet can be found in [vsphere-csi-ds.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-ds.yaml).
+The YAML to deploy `csi-vsphere-node` can be found in [vsphere-csi-node-ds.yaml](https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/csi/vsphere-csi-node-ds.yaml).
 
 Run the following command:
 
 ```bash
-[k8suser@k8master ~]$ kubectl create -f vsphere-csi-ds.yaml
+[k8suser@k8master ~]$ kubectl create -f vsphere-csi-node-ds.yaml
+```
+
+#### 7. Create CRDs for CSI driver registration
```

Comment:
This is where I need to read more of the CSI docs. I really thought the sidecars were supposed to take care of this. @vladimirvivien ?

Reply:
They are required because they provide the look up mechanism for the driver. I went as far not installing these CRDs and the driver absolutely doesn't not function without them.

Reply:
Well that's weird -- they are supposed to be optional. Oh well -- we want to make use of them.

Reply:
I only have a portion of the error that I was using to search for the reason... it starts out with:
```
the server could not find the requested resource (get csinodeinfos.csi.storage.k8s.io <the node FQDN>)
```
then spews more junk after it. This error occurs in the driver_registrar sidecar container.